draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -598,6 +607,106 @@ behavior of the sender. Since QUIC does not provide any kind of application
+| Sender Reports | SR | 200 | {{?RFC3550}}  | partly | - NTP timestamps cannot be replaced by QUIC and are required for synchronization (but see note below)<br>- packet and octet counts cannot be provided by QUIC<br>- see below for *RR*s contained in *SR*s |
+| Receiver Reports | RR | 201 | {{?RFC3550}} | possibly | - *Fration Lost*/*Cumulative Lost*/*Highest Sequence Number received* can directly be inferred from QUIC ACKs<br>- *Interarrival Jitter*/*Last SR* need a QUIC timestamp extension. Using QUIC ts is slightly different because it ignores transmission offsets from RTP timestamps, but that seems like a good thing (see *IJ* above) |
+| Source description | SDES | 202 | {{?RFC3550}}  | no | |
+| Goodbye | BYE | 203 | {{?RFC3550}}  | possibly | using QUIC CONNECTION_CLOSE frame? |
+| Application-defined | APP | 204 | {{?RFC3550}}  | no | |
+| Generic RTP Feedback | RTPFB | 205 | {{?RFC4585}} | partly | see table below |
+| Payload-specific | PSFB | 205 | {{?RFC4585}}  | | see table below |
+| extended report | XR | 207 | {{?RFC3611}} | partly | see table below |
+| AVB RTCP packet | AVB | | |
+| Receiver Summary Information | RSI | | 209 | {{?RFC5760}}
+| Port Mapping | TOKEN | 210 | {{?RFC6284}}  | no? | |
+| IDMS Settings | IDMS | 211 | {{?RFC7272}}  | no | |
+| Reporting Group Reporting Sources | RGRS | 212 | {{?RFC8861}} | |
+| Splicing Notification Message | SNM | 213 | {{?RFC8286}} | no | |
+
+### Notes
+
+* *SR* NTP timestamps: We cannot send NTP timestamps in the same format the SRs use, but couldn't a QUIC timestamp extension provide the same information?
+
+## Generic RTP Feedback (RTPFB) {#generic-feedback}
+
+| Name     | Long Name | Document | Replaced by QUIC | Comments |
+| -------- | --------- | -------- | ---------------- | -------- |
+| Generic NACK | Generic negative acknowledgement | {{?RFC4585}}   | possibly | Using QUIC ACKs |
+| TMMBR | Temporary Maximum Media Stream Bit Rate Request | {{?RFC5104}}  | no | |
+| TMMBN | Temporary Maximum Media Stream Bit Rate Notification | {{?RFC5104}}   | no | |
+| RTCP-SR-REQ | RTCP Rapid Resynchronisation Request | {{?RFC6051}}  | no | |
+| RAMS | Rapid Acquisition of Multicast Sessions | {{?RFC6285}} | no | |
+| TLLEI | Transport-Layer Third-Party Loss Early Indication | {{?RFC6642}}  | no? | no way of telling QUIC peer "don't ask for retransmission", but QUIC would not ask that anyway, only RTCP NACK? |
+| RTCP-ECN-FB | RTCP ECN Feedback | {{?RFC6679}} | partly | QUIC does not provide info about duplicates |
+| PAUSE-RESUME | Media Pause/Resume | {{?RFC7728}} | no | |
+| DBI | Delay Budget Information (DBI) | {{3GPP-TS-26.114}} | |
+| CCFB | RTP Congestion Control Feedback | {{?RFC8888}} | possibly | - *ECN*/*ACK* natively in QUIC<br>- timestamps require QUIC timestamp extension |
+
+## Payload-specific RTP Feedback (PSFB) {#payload-specific-feedback}
+
+| Name     | Long Name | Document | Replaced by QUIC | Comments |
```

Comment:
I think this is a good suggestion - I added "Because QUIC is a generic transport protocol, QUIC cannot replace the following Payload-specific RTP Feedback (PSFB) feedback" at the beginning of the section, but I left the list of feedback types and their documents in the text, just to make what we considered, clear to the reader. 

Does that make sense?

Reply:
Yes, that sounds good. Thanks!